src/Common/src/Interop/Ole32/Interop.IDLFLAGS.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+internal static partial class Interop
+{
+    internal static partial class Ole32
+    {
+        [Flags]
+        public enum IDLFLAGS : ushort
```

Comment:
I don't believe this is correct, it seems to be a dup of both `FUNCFLAGS` and `VARFLAGS`.

oaidl.h has the following definitions for `IDLFLAG`s:
![image](https://user-images.githubusercontent.com/4403806/65746687-d3dbcf80-e142-11e9-98d5-8903c5c9a0d5.png)
The prefix is also in a singular form (it took me awhile to discover the original definitions because of it):
```suggestion
        public enum IDLFLAG : ushort
```

Also there is an obsolete definition called [`IDLFLAGS`](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.idlflag?view=netframework-4.8) in System.Runtime.InteropServices namespace, but it is completely different from the provided here.